crates/ruff_linter/src/rules/flake8_simplify/rules/ast_ifexp.rs

```diff
@@ -50,9 +50,9 @@ impl Violation for IfExprWithTrueFalse {
     fn fix_title(&self) -> Option<String> {
         let IfExprWithTrueFalse { is_compare } = self;
         if *is_compare {
-            Some(format!("Remove unnecessary `True if ... else False`"))
+            Some("Remove unnecessary `True if ... else False`".to_string())
         } else {
-            Some(format!("Replace with `bool(...)"))
+            Some("Replace with `bool(...)".to_string())
         }
     }
```

Comment:
similarly here, we could probably do something like

```rs
    fn fix_title(&self) -> Option<String> {
        let title = if self.is_compare {
            "Remove unnecessary `True if ... else False`"
        } else {
            "Replace with `bool(...)"
        };
        Some(title.to_string())
    }
```

which I would find marginally more readable